homeassistant/components/media_player/webostv.py

```diff
@@ -12,31 +12,36 @@
 
 import homeassistant.util as util
 from homeassistant.components.media_player import (
+    SUPPORT_TURN_ON, SUPPORT_TURN_OFF,
     SUPPORT_NEXT_TRACK, SUPPORT_PAUSE, SUPPORT_PREVIOUS_TRACK,
-    SUPPORT_TURN_OFF, SUPPORT_VOLUME_MUTE, SUPPORT_VOLUME_STEP,
+    SUPPORT_VOLUME_MUTE, SUPPORT_VOLUME_STEP,
     SUPPORT_SELECT_SOURCE, SUPPORT_PLAY_MEDIA, MEDIA_TYPE_CHANNEL,
     MediaPlayerDevice, PLATFORM_SCHEMA)
 from homeassistant.const import (
-    CONF_HOST, CONF_CUSTOMIZE, STATE_OFF, STATE_PLAYING, STATE_PAUSED,
+    CONF_HOST, CONF_CUSTOMIZE, STATE_OFF,
+    STATE_PLAYING, STATE_PAUSED,
     STATE_UNKNOWN, CONF_NAME)
 from homeassistant.loader import get_component
 import homeassistant.helpers.config_validation as cv
 
 REQUIREMENTS = ['https://github.com/TheRealLink/pylgtv'
                 '/archive/v0.1.2.zip'
-                '#pylgtv==0.1.2']
+                '#pylgtv==0.1.2',
+                'websockets==3.2',
+                'wakeonlan==0.2.2']
 
-_CONFIGURING = {}
+_CONFIGURING = {}  # type: Dict[str, str]
 _LOGGER = logging.getLogger(__name__)
 
+CONF_MAC = 'mac'
```

Comment:
Thanks, I missed that because I started with `CONF_MAC_ADDRESS` like switch/wake_on_lan. Will fix.